err_fmt_test.go

```diff
@@ -857,6 +976,321 @@ func TestFmt(t *testing.T) {
+			fn:     topWrap,
+			source: errStd,
+			expect: expect{
+				v:    "top wrap: mid wrap: bottom wrap: an error",
+				hash: "top wrap: mid wrap: bottom wrap: an error",
+				s:    "top wrap: mid wrap: bottom wrap: an error",
+				q:    `"top wrap": "mid wrap": "bottom wrap": "an error"`,
+				plus: plusRE(
+					`an error\n`, "",
+					`bottom wrap\n`, `err_fmt_test.go:\d+\n`,
+					`mid wrap\n`, `err_fmt_test.go:\d+\n`,
+					`top wrap\n`, `err_fmt_test.go:\d+$`,
+				),
+			},
+		},
+		{
+			name:   "clues.Wrap errors.New",
+			fn:     topWrap,
+			source: errErrs,
+			expect: expect{
+				v:    "top wrap: mid wrap: bottom wrap: an error",
+				hash: "top wrap: mid wrap: bottom wrap: an error",
+				s:    "top wrap: mid wrap: bottom wrap: an error",
+				q:    `"top wrap": "mid wrap": "bottom wrap": "an error"`,
+				plus: plusRE(
+					`an error\n`, "",
+					`github.com/alcionai/clues_test.init\n`, `err_fmt_test.go:\d+\n`,
+					`runtime.doInit\n`, `proc.go:\d+\n`,
+					`runtime.doInit\n`, `proc.go:\d+\n`,
+					`runtime.main\n`, `proc.go:\d+\n`,
+					`runtime.goexit\n`, `runtime/.*:\d+\n`,
+					`bottom wrap\n`, `err_fmt_test.go:\d+\n`,
+					`mid wrap\n`, `err_fmt_test.go:\d+\n`,
+					`top wrap\n`, `err_fmt_test.go:\d+$`,
+				),
+			},
+		},
+		{
+			name:   "clues.Wrap fmt.Errorf",
+			fn:     topWrap,
+			source: fmtErrf,
+			expect: expect{
+				v:    "top wrap: mid wrap: bottom wrap: an error",
+				hash: "top wrap: mid wrap: bottom wrap: an error",
+				s:    "top wrap: mid wrap: bottom wrap: an error",
+				q:    `"top wrap": "mid wrap": "bottom wrap": "an error"`,
+				plus: plusRE(
+					`an error\n`, "",
+					`bottom wrap\n`, `err_fmt_test.go:\d+\n`,
+					`mid wrap\n`, `err_fmt_test.go:\d+\n`,
+					`top wrap\n`, `err_fmt_test.go:\d+$`,
+				),
+			},
+		},
+		{
+			name:   "clues.Wrap clues.New",
+			fn:     topWrap,
+			source: cluErr,
+			expect: expect{
+				v:    "top wrap: mid wrap: bottom wrap: an error",
+				hash: "top wrap: mid wrap: bottom wrap: an error",
+				s:    "top wrap: mid wrap: bottom wrap: an error",
+				q:    `"top wrap": "mid wrap": "bottom wrap": "an error"`,
+				plus: plusRE(
+					"an error\n", `err_fmt_test.go:\d+\n`,
+					`bottom wrap\n`, `err_fmt_test.go:\d+\n`,
+					`mid wrap\n`, `err_fmt_test.go:\d+\n`,
+					`top wrap\n`, `err_fmt_test.go:\d+$`,
+				),
+			},
+		},
+		// ---------------------------------------------------------------------------
+		// plain stacked
+		// ---------------------------------------------------------------------------
+		{
+			name:   "clues.PlainStack stderr.New",
+			fn:     topPlainStack,
+			source: errStd,
+			expect: expect{
+				v:    "an error",
+				hash: "an error",
+				s:    "an error",
+				q:    `"an error"`,
+				plus: plusRE(
+					`an error\n`, "",
+					"", `github.com/alcionai/clues/err_fmt_test.go:\d+\n`,
+					"", `github.com/alcionai/clues/err_fmt_test.go:\d+\n`,
+					"", `github.com/alcionai/clues/err_fmt_test.go:\d+$`,
```

Comment:
I think you added too much path prefix info on these
```suggestion
					"", `/err_fmt_test.go:\d+\n`,
					"", `/err_fmt_test.go:\d+\n`,
					"", `/err_fmt_test.go:\d+$`,
```